CONTRIBUTING.adoc

```diff
@@ -1,14 +1,14 @@
 = CONTRIBUTING
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+:toc:
 
 :uri-oracle-oca: https://www.oracle.com/technetwork/community/oca-486395.html
```

Comment:
Please change this to our new online OCA which is now live: https://oca.opensource.oracle.com